zenko-2018-07-11-xml.api.json

```diff
@@ -107,6 +107,92 @@
                     "EncodingType": {}
                 }
             }
+        },
+        "searchBucketVersions": {
+            "http": {
+                "method": "GET",
+                "requestUri": "/{Bucket}"
+            },
+            "input": {
+                "type": "structure",
+                "required": [
+                    "Bucket",
+                    "Query"
+                ],
+                "members": {
+                    "Query": {
+                        "location": "querystring",
+                        "locationName": "search"
+                    },
+                    "Bucket": {
+                        "location": "uri",
+                        "locationName": "Bucket"
+                    },
+                    "Delimiter": {
+                        "location": "querystring",
+                        "locationName": "delimiter"
+                    },
+                    "EncodingType": {
+                        "location": "querystring",
+                        "locationName": "encoding-type"
+                    },
+                    "KeyMarker": {
+                        "location": "querystring",
+                        "locationName": "key-marker"
+                    },
+                    "MaxKeys": {
+                        "location": "querystring",
+                        "locationName": "max-keys",
+                        "type": "integer"
+                    },
+                    "Prefix": {
+                        "location": "querystring",
+                        "locationName": "prefix"
+                    },
+                    "ExpectedBucketOwner": {
+                        "location": "header",
+                        "locationName": "x-amz-expected-bucket-owner"
+                    },
+                    "Versions": {
+                        "location": "querystring",
+                        "locationName": "versions",
+                        "type": "boolean"
+                    },
```

Comment:
I find it a bit strange to have both an Api name and a flag versions. What happens if we call `searchBucketVersions` with version being `false` ?